README.md

```diff
@@ -1,63 +1,172 @@
+   ```
+   npm update && npm install
+   npm run build
+   ```
 
-## Testing
+3. To connect to the axelar network, run `npm install axelarnetwork/axelar-local-dev`.
 
-All examples can be tested with `node scripts/test/{example-name} ${local|testnet} ...params`. See below for what values params can take. All params are optional and have default values and can be ommited.
+4. To run a local node, open `axelar-local-gmp-examples` in a separate terminal and run `node scripts/createLocal`. You’ll need to have this node running to deploy the dapps.
 
-### Sent token
+   The five supported networks are deployed: Moonbeam (ChainID: 2500), Avalanche (chainID: 2501), Fantom (chainID: 2502), Ethereum (chainID: 2503), and Polygon (chainID: 2504) on port 8500.
 
-Run `node scripts/test examples/send-token ${local|testnet} ${source-chain} ${destination-chain} ${amount}` to send aUSDC from the source to the destination. To run on testnet you need to fund `0xBa86A5719722B02a5D5e388999C25f3333c7A9fb` with aUSDC and replace `local` with `testnet`.
+5. To make sure that the address we use for examples is funded on all five supported testnets, run `node scripts/checkBalances`. We use `0xBa86A5719722B02a5D5e388999C25f3333c7A9fb` to deploy and test all examples. Alternatively, you can use `--address [<address>]` to specify a different address.
 
-### Deposit Address
+## Deploy and test each example
 
-Run `node scripts/test examples/deposti-address ${local|testnet} ${source-chain} ${destination-chain} ${amount}` to send aUSDC from the source to the destination. To run on testnet you need to fund `0xBa86A5719722B02a5D5e388999C25f3333c7A9fb` with aUSDC and replace `local` with `testnet`.
+For each example, you can allow the defaults to run or enter:
+
+- `local` or `testnet` for both the test and the deploy steps, except for deposit-address and send-token, which run on testnet.
+
+- `source-chain` and `destination-chain`
+
+\[Editor: What's the correct way to identify each? by chain-ID? as a "string" or integer?]
+
+- variables such as `message`, `amount`, and `account`
+
+\[Editor: What is the expected syntax and data type: message: "message"?, amount: integer?, account: "string" or integer? What are the defaults for each?]
+
+\[Editor: In regard to this phrase: "All params are optional and have default values and can be ommited." I think the writer means it in the sense that it's optional for the user to enter a value bc there are defaults. In API, the convention is to call variables with default values "required". I've reworded this to avoid this ambiguity.]
+
+and run the local deploy and test.
 
 ### Call contract
 
-Run `node scripts/test examples/call-contract ${local|testnet} ${source-chain} ${destination-chain} ${message}` to relay a message from `source-chain` to `destination-chain`.
+This dapp relays a message from source-chain to destination-chain.
+
+1. To deploy the dapp, run::
+
+`node scripts/deploy examples/call-contract [<local|testnet>]`
```

Comment:
@ggutoski  I was able to execute this by entering `node scripts/deploy examples/call-contract local` but I thought there were defaults. What is the default? local? and how do I enter it to just use the default? I tried `node scripts/deploy examples/call-contract ${local|testnet}` but it asked to provide a value for the variable.

Reply:
I believe there are no defaults.  You must type either `local` or `testnet`